neo4j_utils/_n4jversion.py

```diff
@@ -11,7 +12,7 @@ class Neo4jVersion:
     Provides version information for Neo4j.
     """
 
-    major: int | None
+    major: Union[int, None]
```

Comment:
bc this is a class attribute, I would expect `from __future__ import annotations` to work